src/editor/user-tasks/task-definitions.js

```diff
@@ -1,27 +1,17 @@
 import UserTask from "./user-task";
 import ExternalLinkTask from "./external-link-task";
 import { store } from "../store";
-import * as actions from "../actions/actions";
+import {
+  moveCursorToNextChunk,
+  toggleWrapInEditors
+} from "../actions/editor-actions";
 import { evaluateText, evaluateNotebook } from "../actions/eval-actions";
 import {
   toggleFileModal,
   toggleHelpModal,
   toggleHistoryModal
 } from "../actions/modal-actions";
-
-// FIXME: remove requirement to import store in this file by attaching
```

Comment:
please retain this FIXME

Reply:
This FIXME looks wrong though. There is no doubt a way to avoid importing store in this file (if that really does matter that much), but I don't think it has anything to do with keypress handling.

Reply:
it does, the keypress handler needs access to the store to be able to dispatch action.just trust me on this -- tasks are a bit of a tangle right now, and i just added that FIXME like last week to give myself a breadcrumb :-)

Reply:
Well, ok, I can add it back although I'm still confused about what is meant here and I think I understand this code about as well as anyone except for you at this point. In general I would prefer if we aimed to have the codebase be readable by anyone, not just by one person with specific context or an idea of what they want to do next.

Reply:
i understand that concern, but as always, there are many tradeoffs. the truth of the matter is that unless/until we get more help, i'm doing the majority of the client-side engineering, and context switching to write out everything as an issue or switching branches to fix something that i notice while fixing an unrelated issue slows me down a lot and disrupts my thinking process. we've been using FIXMEs this way for a long time to capture these thoughts in a less disruptive way, and for now i'm comfortable with the way this tradeoff balances -- i think FIXME comments like this are valuable to me and hamilton for leaving notes in context, and that they can be easily ignored by folks for whom they are not relevant. but your concern is noted, hopefully we'll be able to improve on this over time :-)

Reply:
Yeah, I definitely have concerns about whether the big bang PRs that have been common up to now will scale as we add more contributors. I'd rather get people used to making smaller and more focused PRs, and filing issues as appropriate. But we can save that conversation for a later time, for now I've added this back.